example/.gitignore

```diff
@@ -1,8 +1,10 @@
 .serverless
+.serverless_plugins
 /node_modules/
 
 # Ignore ignitor
 ignitor
+.output
```

Comment:
the new invoke command needs to have an output file in order for it to work (strange I know)